modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/SubscribeManager.kt

```diff
@@ -63,12 +64,17 @@ class SubscribeManager @Inject constructor(
 
     data class PodcastSubscribe(val podcastUuid: String, val sync: Boolean)
 
+    @SuppressLint("CheckResult")
     private fun setupSubscribeRelay(): PublishRelay<PodcastSubscribe> {
         val source = PublishRelay.create<PodcastSubscribe>()
         source
             .observeOn(Schedulers.io())
             .doOnNext { info -> Timber.i("Adding podcast to addPodcast queue ${info.podcastUuid}") }
-            .flatMap({ info -> addPodcast(info.podcastUuid, sync = info.sync, subscribed = true).toObservable() }, true, 5)
+            .flatMap({ info ->
+                // shouldAutoDownload = true because the user manually subscribed to the podcast,
+                // so we want to automatically download episodes at this moment.
```

Comment:
When I imported podcast with an OPML file all of the podcast had the auto download enabled with limit of 2 episodes.

Reply:
Yeah. I am going to handle this issue in another PR. See here: https://github.com/Automattic/pocket-casts-android/issues/2998